src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -314,6 +315,33 @@ public Job call() throws SaltStackException {
     }
 
     /**
+     * Get previously run jobs.
+     * @return map containing run jobs keyed by job id.
+     * @throws SaltStackException if anything goes wrong
+     */
+    public Map<String, Job> getJobs() throws SaltStackException {
+         Result<List<Map<String, Job>>> result = connectionFactory
+                .create("/jobs", JsonParser.JOBS, config)
+                .getResult();
+         return result.getResult().get(0);
```

Comment:
Yes, I agree - getting the 1st element is ugly. I only followed this "pattern" because I saw it in different methods.

BTW - do you know if there is some declarative description of salt api (e.g. xsd file)? Or there is only python code in salt and the docs (which only describe couple of examples)? Having full api description in one place would be really terrific.